ouroboros-consensus/src/Ouroboros/Consensus/HardFork/History/Summary.hs

```diff
@@ -64,10 +66,10 @@ import           Cardano.Binary (enforceSize)
 
 import           Ouroboros.Consensus.Block
 import           Ouroboros.Consensus.BlockchainTime.WallClock.Types
-import           Ouroboros.Consensus.Util.Counting
-
 import           Ouroboros.Consensus.HardFork.History.EraParams
 import           Ouroboros.Consensus.HardFork.History.Util
+import           Ouroboros.Consensus.Node.Serialisation
```

Comment:
Will fix.

Is there some secret to knowing which package each import is from? I find my self doing a lot of text search to try and figure out where to put each new import statement.

Reply:
See bullet point 15 "Import lists" in https://github.com/input-output-hk/ouroboros-network/blob/master/ouroboros-consensus/docs/StyleGuide.md

That's the most indicative explanation. But eg it doesn't mention the HardFork thing; I'm only aware of that because of having seen it set apart in the existing code.